src/ds5/ds5-auto-calibration.cpp

```diff
@@ -28,6 +28,24 @@ namespace librealsense
         float rightRotation[9]; // Right rotation
     };
 
+    struct TareCalibrationResult
+    {
+      uint16_t status;  // DscStatus
+      uint32_t tareDepth;  // Tare depth in 1/100 of depth unit
+      uint32_t aveDepth;  // Average depth in 1/100 of depth unit
+      int32_t curPx;    // Current Px in 1/1000000 of normalized unit
+      int32_t calPx;    // Calibrated Px in 1/1000000 of normalized unit
+      float curRightRotation[9]; // Current right rotation
+      float calRightRotation[9]; // Calibrated right rotation
+      uint16_t accuracyLevel;  // [0-3] (Very High/High/Medium/Low)
+      uint16_t iterations;        // Number of iterations it took to converge
+      //int32_t errors[iterations];  // Array of errors in 1/1000000 of a percent
+      //int32_t x[iterations];    // Intrinsic scan: array of Px in 1/1000000 normalized unit
+      //                         // Extrinsic scan: array of Ry in 1/100000 radian
+      //float beforeHealthCheck; // Before health check number
+      //float afterHealthCheck;  // After health check number
+    };
+
```

Comment:
Verify it is handled with `#pragma pack(push, 1)`

Reply:
It is in a section begins with "#pragma pack(push, 1)" and ends with "#pragma pack(pop)"